### DISCUSSION TITLE
Trigger popup message text for modified local achievements is misleading.

### DISCUSSION THREAD

**Discussion by redwizard42** [RetroAchievements/RAIntegration#968] [Completed]
Just had a junior think their unsaved local achievement wasn't working because they saw the 'NOT unlocked' part of "Modified local achievement NOT unlocked." popup and thought that meant it didn't trigger.  That and the NOT unlocked due to modified RAM texts are a tiny bit misleading.  While it's not critical, I think a less negative wording could be used. While a little education for the Jr straightened it out, some small rewording might be useful for clarity.

**Comment by Jamiras** [comment]
Implemented in #975
